src/components/InputDropdown/index.js

```diff
@@ -1,80 +1,60 @@
 /** @jsx jsx */
-import { jsx } from '@emotion/core'
-import React from 'react'
-import Text from 'components/Text'
-import { ReactComponent as Chevron } from 'static/icons/chevron.svg'
+import { jsx } from '@emotion/core';
+import React from 'react';
+import { ReactComponent as Chevron } from 'static/icons/chevron.svg';
 
-import styles from './InputDropdown.styles'
+import styles from './InputDropdown.styles';
+
+const DEFAULT = 'placeholder';
 
 class InputDropdown extends React.Component {
   constructor(props) {
-    super(props)
-    this.listEl = null
-    this.inputEl = null
+    super(props);
     this.state = {
-      isOpen: false,
-      value: '',
-    }
-  }
+      value: DEFAULT,
+    };
 
-  componentDidMount() {
-    window.addEventListener('mousedown', this.handleClick)
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Given that `handleChange` is an arrow function I don't think we need this binding. If we move `this.state` to the top level we can get rid of the constructor all together 🔥

Reply:
Actually @WesSouza hooks PR will take care of most of this. Never mind!